test/controllers/colonies_controller_test.rb

```diff
@@ -29,6 +29,7 @@ def setup
   end
 
   test 'creates with valid attributes and redirects html' do
+    binding.pry
```

Comment:
should probably take this out before we merge - I don't know would happen when travis runs the specs